content/en/blog/_posts/2024-XX-XX-websocket-transition.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: blog
+title: 'Streaming Transitions from SPDY to WebSockets'
+date: 2024-XX-XX
+slug: websockets-transition
+author: >
+  [Sean Sullivan](https://github.com/seans3) (Google)
+---
+
+With Kubernetes 1.31, PortForward over Websockets is moving to beta.
+RemoteCommand over Websockets previously graduated to beta in v1.30.
+
+Today’s article is about streaming API’s and the protocols used to implement
+these API’s.
+
+## Why have we modified the streaming protocol ?
+
+Consider the following situation: you’ve configured a proxy or gateway in front
+of your Kubernetes cluster, but you’ve discovered four kubectl commands no
```

Comment:
do we need to clarify that the proxy or gateway is in fron of the kubernetes cluster control-plane/apiserver?

Reply:
This entire paragraph has been re-written. Please have a look.